EventsExpress/Mapping/LocationMapperProfile.cs

```diff
@@ -10,6 +10,9 @@ public LocationMapperProfile()
         {
             CreateMap<LocationDto, EventLocation>().ReverseMap();
             CreateMap<EventDto, LocationDto>()
```

Comment:
I guess we don't need this now

Reply:
CreateMap<LocationDto, EventLocation>().ReverseMap() is used in LocationService 
CreateMap<EventDto, LocationDto>() is used in EventService

Reply:
done